terraform/modules/aws/rds/main.tf

```diff
@@ -0,0 +1,53 @@
+#create security group for the database
+
+resource "aws_security_group" "database_security_group" {
+  name        = "database security group"
+  description = "enable postgres access on port 5432"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description = "rds private subnet access"
+    from_port   = var.rds_sg.ingress.from_port
+    to_port     = var.rds_sg.ingress.to_port
+    protocol    = var.rds_sg.ingress.protocol
+    cidr_blocks = var.rds_sg.ingress.cidr_blocks
+  }
+
+  #egress {
+  #from_port        = var.port
+  #to_port          = var.port
+  #protocol         = var.protocol
+  #cidr_blocks      = var.cidr
+  #}
+
+  tags = {
+    Name = var.sg_name
+  }
+}
+
+
+# create the subnet group for the rds instance
+resource "aws_db_subnet_group" "database_subnet_group" {
+  name       = var.db_subnet_name
+  subnet_ids = var.rds_subnet_ids
+
+
+  tags = {
+    Name = "rds-subnet-tf"
+  }
+}
+
+# create the rds instance
+resource "aws_db_instance" "omega_db" {
```

Comment:
Please attach security group that you created to the instance

Reply:
Done